up-and-running/platforms/aws-cloud-formation-template.md

```diff
@@ -0,0 +1,323 @@
+      id: ami-6c101b0a
+    eu-west-2:
+      id: ami-056d7a61
+    sa-east-1:
+      id: ami-4bd8b727
+    us-east-1:
+      id: ami-20631a36
+    us-east-2:
+      id: ami-a5b196c0
+    us-gov-west-1:
+      id: ami-ff22a79e
+    us-west-1:
+      id: ami-9fe6c7ff
+    us-west-2:
+      id: ami-45224425
+Resources:
+  EC2Instance:
+    Type: 'AWS::EC2::Instance'
+    Properties:
+      EbsOptimized: !FindInMap
+        - EnvironmentSettings
+        - !Ref Environment
+        - EbsOptimized
+      InstanceType: !FindInMap
+        - EnvironmentSettings
+        - !Ref Environment
+        - InstanceType
+      ImageId: !FindInMap
+        - AWSRegionToAMI
+        - !Ref AWS::Region
+        - id
+      KeyName: !Ref SSHKeyName
+      SecurityGroups:
+        - !Ref InstanceSecurityGroup
+      UserData:
+        Fn::Base64: !Sub |
+          #!/bin/bash -x
+          parted /dev/xvdb mklabel msdos
+          parted -a opt /dev/xvdb mkpart primary ext4 0% 100%
+          mkfs.ext4 -L datapartition /dev/xvdb1
+          mount -o defaults /dev/xvdb1 /mnt
+          mkdir -p /mnt/docker
+          apt-get update
+          apt-get install -y python-pip
+          pip install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz
+          cfn-init -v --resource EC2Instance --stack "${AWS::StackName}" --region "${AWS::Region}"
+          cfn-signal -e $? --resource EC2Instance --stack "${AWS::StackName}" --region "${AWS::Region}"
+    Metadata:
+      'AWS::CloudFormation::Init':
+        config:
+          files:
+            /home/ubuntu/npme-setup/npme-settings.json:
+              content: !Join
+                - ''
+                - - '{'
+                  - '  "allowpublishes": { "value": "allowpublishes_all" },'
+                  - '  "auth_source": { "value": "auth_type_open" },'
+                  - '  "authfetch": { "value": "authfetch_no" },'
+                  - '  "authglobal": { "value": "authglobal_no" },'
+                  - '  "authwww": { "value": "authwww_no" },'
+                  - '  "authz_cache_enabled": { "value": "authz_cache_enabled_yes" },'
+                  - '  "branding": { "value": "'
+                  - !Ref Branding
+                  - '" },'
+                  - '  "couch_url_remote": { "value": "'
+                  - !FindInMap
+                    - EnvironmentSettings
+                    - !Ref Environment
+                    - UpstreamUrl
+                  - '" },'
+                  - '  "couchdb_host_path": { "value": "/mnt/couch" },'
+                  - '  "data_host_path": { "value": "/mnt/data" },'
+                  - '  "es_host_path": { "value": "/mnt/es" },'
+                  - '  "packages_host_path": { "value": "/mnt/packages" },'
+                  - '  "postgres_host_path": { "value": "/mnt/pg" },'
+                  - '  "read_through_cache": { "value": "read_through_cache_yes" },'
+                  - '  "redis_host_path": { "value": "mnt/redis" },'
```

Comment:
Need to update this to match the correction I had to make to the gist earlier, the redis host path value should be `/mnt/redis`.